h2o-genmodel-extensions/xgboost/build.gradle

```diff
@@ -5,16 +5,18 @@ description = "H2O GenModel XGBoost support"
 dependencies {
     compile project(":h2o-genmodel")
 
-    // XGBoost dependencies but not officially published
-    compile files("xgblib/xgboost4j-0.7.1.jar",
-                  "xgblib/xgboost4j-native-osx_minimal-0.7.1.jar",
-                  "xgblib/xgboost4j-native-linux_minimal-0.7.1.jar",
-                  "xgblib/xgboost4j-native-linux_ompv3-0.7.1.jar",
-                  "xgblib/xgboost4j-native-linux_gpuv4-0.7.1.jar")
-
-    // These are dependencies extracted from xgboost/jvm-packages/pom.xml
-    compile 'com.esotericsoftware.kryo:kryo:2.21'
-    compile 'commons-logging:commons-logging:1.2'
+    // XGBoost dependencies published into Maven central by H2O
+    // Versioning rules may differ for XGBoost artifacts published by H2O
+    compile('ai.h2o:xgboost4j:0.7.3') {
```

Comment:
Can we use 'transitive = false' to exclude all transitive dependencies? Is there a transitive dependency we want to include?

Reply:
There are two transitive dependencies included. You can see them removed above. Those were manually included before this change:

```groovy
compile 'com.esotericsoftware.kryo:kryo:2.21'
compile 'commons-logging:commons-logging:1.2'
```